backend/frontend/tests/test_claims_page_button.py

```diff
@@ -0,0 +1,41 @@
+from project_review_coordination.models import ProjectReviewBundleClaim
+from core.tests.factories import UserFactory
+from .frontend_test_mixin import FrontendTestMixin
+from playwright.sync_api import expect
+
+
+class TestClaimButtons(FrontendTestMixin):
+    def setUp(self):
+        super().setUp()
+        self.user = UserFactory(
+            email="staff_1@umuzi.org",
+            is_staff=True,
+        )
+        self.user.set_password(self.user.email)
+        self.user.save()
+
+        self.user_2 = UserFactory(email="staff_2@umuzi.org", is_staff=True)
+        self.user_2.set_password(self.user_2.email)
+        self.user_2.save()
```

Comment:
Almost there...

There's no need for user_2 to:
- be in `setU`p since it will only be used in the second test
- it should, therefore, also not be attached to `self`

The difference between the 2 users is that we need to be logged in as one user before running all the tests and the first test makes use of the logged in user